lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -931,7 +931,7 @@ def _is_installed(cmd):
 
         if _is_installed("systemctl"):
             systemd_magmad_running = self.exec_command_output(
-                "systemctl is-active magma@magmad",
+                "systemctl is-active magma@magmad || /bin/true",
```

Comment:
This can be remove once [this](https://github.com/magma/magma/pull/14034 ) is merged

Reply:
If this PR gets merged first, there are chances we will miss this. Please add a comment here in this PR change.

Btw do we really need this change? Because if services are available, this will always return the actual service status and if not then always `"inactive"` and we don't rely on command exit status here.

Reply:
The other PR was merged, so this can be removed again.

Reply:
Hotfix was reverted.